guidance/los_guidance/los_guidance/los_guidance_node.py

```diff
@@ -0,0 +1,87 @@
+import rclpy
+import numpy as np
+from rclpy.node import Node
+from nav_msgs.msg import Odometry
+from transforms3d.euler import quat2euler, euler2quat
+from geometry_msgs.msg import Point
+from los_guidance.los_guidance import LOSGuidance
+
+class LOSGuidanceNode(Node):
+    def __init__(self):
+        super().__init__("los_guidance_node")
+
+        self.declare_parameters(
+            namespace='',
+            parameters=[
+                ('los_guidance.p0', [0.0, 0.0]),
+                ('los_guidance.p1', [0.0, 0.0]),
+                ('los_guidance.look_ahead_distance', 0.0)
+            ])
+        
+        p0 = self.get_parameter('los_guidance.p0').get_parameter_value().double_array_value
+        p1 = self.get_parameter('los_guidance.p1').get_parameter_value().double_array_value
+        self.look_ahead = self.get_parameter('los_guidance.look_ahead_distance').get_parameter_value().double_value
+
+        self.p_next = np.array([10000.0, 10000.0])
+        
+        self.get_logger().info(f"p0: {p0}")
+        self.get_logger().info(f"p1: {p1}")
+        self.get_logger().info(f"look_ahead_distance: {self.look_ahead}")
+
```

Comment:
Again, test code should be removed and it should be made compatible with the waypoint manager.